tests/unit/test_ochazuke.py

```diff
@@ -128,55 +128,50 @@ def test_needsdiagnosis_invalid_param_values(self, mock_get):
             rv.data.decode())
 
     def test_date_range(self):
-        """Given from_date and to_date, return a number of days."""
+        """Given from_date and to_date, return a list of days."""
         from_date = '2018-01-02'
         to_date = '2018-01-04'
-        days = ['2018-01-02', '2018-01-03']
+        days = ['2018-01-02', '2018-01-03', '2018-01-04']
         self.assertCountEqual(helpers.get_days(from_date, to_date), days)
         self.assertCountEqual(helpers.get_days(to_date, from_date), days)
 
     def test_date_range_invalid(self):
-        """Given from_date and to_date, return a number of days."""
+        """Given an invalid date, return None for the range."""
         from_date = '2018-01-02T23:00'
         to_date = '2018-01-04'
         self.assertEqual(helpers.get_days(from_date, to_date), None)
 
     def test_date_range_same_day(self):
-        """Given from_date and to_date, return a number of days."""
+        """Given a same day range, return the one day range."""
         from_date = '2018-01-02'
         to_date = '2018-01-02'
         self.assertEqual(helpers.get_days(from_date, to_date), ['2018-01-02'])
 
     def test_get_timeline_slice(self):
         """Given a list of dates, return the appropriate slice of data."""
         dates = ['2018-05-16', '2018-05-17']
-        sliced = [
-            {"count": "485", "timestamp": "2018-05-16T02:00:00Z"},
-            {"count": "485", "timestamp": "2018-05-17T03:00:00Z"}
-            ]
+        sliced = [{"count": "485", "timestamp": "2018-05-16T02:00:00Z"},
+                  {"count": "485", "timestamp": "2018-05-17T03:00:00Z"},
```

Comment:
Are these trailing commas (here and on line 162) intentional for a fancy PEP8/linter reason with which I'm not yet familiar (version control prettiness doesn't seem to apply... unless this list might grow, I guess)? 😀 

Or can they be deleted and the closing bracket moved up?

Reply:
Trailing commas are usually nicer for diff. but in this case it doesn't matter. So indeed let me remove them.